node/src/reactor.rs

```diff
@@ -324,6 +331,12 @@ impl Drop for RunnerMetrics {
     }
 }
 
+/// Exit status of a runner instance.
+pub type RunnerExitStatus = Result<(), u8>;
+
+/// Exit code indicating that node should shut down for a scheduled upgrade.
+pub const UPGRADE_EXIT_CODE: u8 = 0;
```

Comment:
I think it might be better to have an enum here, but this already conflicts with changes I have locally, so I'll make that change in my PR if you don't mind?  I assume your planned next steps wouldn't need this changed, so we'll not end up conflicting again here?

Reply:
I didn't use enum on purpose as that would tie, otherwise generic `Reactor/Runner`, to the domain. I thought that generic exit codes were a good compromise.

If you need to make it an enum I won't oppose.